### PR TITLE
Make the package

### LINKED ISSUE #13: Fix packaging
It looks like that Pipfile permits to use the latest versions which resulted in a problem with `simple_salesforce`

```
    self.sf = Salesforce(username=self.cfg["username"],
TypeError: __init__() got an unexpected keyword argument 'sandbox'
```

As it turned out the whole package uses only the latest versions of dependencies.

The future proof solution is to use either `flit` or `poetry`.

### PR DESCRIPTION
**Pull request by 1oglop1**
Fixes #13 
Implements a simple test #14 

### Required change in travis

@giuliocalzolari Could you please generate a PYPI Token of from the project and add it to the travis env as `PYPI_TOKEN`

### CODE REVIEW

**Comment by 1oglop1** [comment]
@giuliocalzolari Tests are passing now, I've noticed the problem with usage of `datetime.date.today()` so I monkey patched  the fixed date for the test.

File: salesforce_timecard/core.py [hunk 1]
```diff
@@ -1,197 +1,263 @@
...
             }
         return rs
 
-    def delete_time_entry(self, id):
+    def delete_time_entry(self, _id):
         try:
-            self.sf.pse__Timecard_Header__c.delete(id)
-        except:
-            logger.error("failed on deletion id:{}".format(id))
+            self.sf.pse__Timecard_Header__c.delete_cmd(_id)
```
Comment (giuliocalzo) [inline salesforce_timecard/core.py:253]: @1oglop1 I believe this is a typo... 

```
self.sf.pse__Timecard_Header__c.delete(_id)
```

**Review by giuliocalzo — CHANGES_REQUESTED** [review]

File: salesforce_timecard/core.py (hunk 1 above)
Comment (1oglop1) [reply salesforce_timecard/core.py:253]: I've added suffix `_cmd` to `list`, `delete`, and `add` due to the `list` was shadowing built-in `list()`.

**Comment by giuliocalzo** [comment]
@1oglop1 it seems the tests are failing https://travis-ci.org/github/giuliocalzolari/salesforce-timecard

**Pull request merged**